### PR TITLE
Issue #103 added gitignore and header to miopen.h 

### PR DESCRIPTION
**Pull request by dereklstinson**
I added a gitignore to the build folder and my .vscode folder.  Also, I added #include<stdbool.h> to make it compatible with gcc.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+build/
```
Comment (daniellowell) [inline .gitignore]: It is totally arbitrary what a user will call their build folder. I don't expect we can add a specific user environment directories into our repo.

Comment (pfultz2) [reply .gitignore]: If the user wants git to ignore the build directory its best not to place the build directory in the source directory.

File: include/miopen/miopen.h [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 #include <miopen/config.h>
 #include <miopen/export.h>
-
+#include <stdbool.h>
```
Comment (daniellowell) [inline include/miopen/miopen.h]: @pfultz2 Any objections?

**Comment by daniellowell** [comment]
This has been added to MIOpen 2.0.0

**Pull request closed without merging**